pkg/controller/releasemanager/plan/deleteTaggedServiceLevels.go

```diff
@@ -0,0 +1,51 @@
+package plan
```

Comment:
These delete plans are almost identical, we should probably come up with some abstraction.

Reply:
:homer_disappear: